src/rulesets/reader.ts

```diff
@@ -1,53 +1,47 @@
-import { merge } from 'lodash';
+import { parse } from '@stoplight/yaml';
 import { readParsable } from '../fs/reader';
 import { RuleCollection } from '../types';
-import { IRulesetFile } from '../types/ruleset';
-import { formatAjv } from './ajv';
```

Comment:
if this file is no longer being used can we remove it?

Reply:
The file is still used! :)